IFC4x3/Properties/c/ConstructionCompany_3q5el6dSLCw8hGDPwFONxo/DocProperty.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="ConstructionCompany_3q5el6dSLCw8hGDPwFONxo" Name="ConstructionCompany" UniqueId="f4168bc6-9dc5-4ce8-8ad0-359e8f617ef2" PropertyType="p_enumeratedvalue" PrimaryDataType="IfcLabel">
+	<Enumeration xsi:type="DocPropertyEnumeration" xsi:nil="true" href="PEnum_Created" />
+</DocProperty>
+
```

Comment:
Mixing description of the product with description of the process (e.g. who performed the process of construction) does not feel entirely clean to me. I think that is sort of shortcutting the structure with IfcProcess, IfcProduct and IfcResource.

Reply:
@czapplitec Could use IfcOrganization for this.

Reply:
Done. a14c639b6c4248c34ffadf5422b6425f466a4277